index.html

```diff
@@ -24,6 +24,21 @@
         company: "Dynatrace",
         companyURL: "https://dynatrace.com",
         w3cid: "48276"
+      },
+      {
+        name: "Bogdan Drutu",
+        company: "Google",
+        companyURL: "https://google.com"
+        //TODO: w3cid is missing
+      },
+      {
+        name: "Nik Molnar",
+        company: "Microsoft",
+        companyURL: "https://microsoft.com"
+        w3cid: "104238"
+      },
+      {
+        name: "Yuri Shkuro" //TODO: as invited expert
```

Comment:
@plehegar please help here

Reply:
1. Go to https://www.w3.org/Member/ - use your credentials to login
1. Then on the right side press User Account
1. A personal page will open, look in the URL `https://www.w3.org/users/XXXXXX` where `XXXXXX` will be your w3cid

Reply:
Cc: @yurishkuro

Reply:
My id is https://www.w3.org/users/104074